.gitignore

```diff
@@ -50,6 +50,8 @@ dist/
 # mock
 mock/index.d.ts
 mock/index.js
+mock/jest.d.ts
+mock/jest.js
```

Comment:
Curious, why did we add it to `.gitignore`?

Reply:
Because I build them (see the `package.json` file). This way, pure javascript codebase (without jest configured to use babel TS preset) will be able to resolve the mock.